### DISCUSSION TITLE
Support AnimeBytes

### DISCUSSION THREAD

**Discussion by BoBeR182** [RogueOneEcho/caesura#88] [Ideas]
AnimeBytes has a better.php that could benefit from this tool.
Limitations on making sure all RED or OPS torrents don't auto upload and only ones from AB are transcoded and uploaded since they have stricter upload rules regarding music.

**Comment by RogueOneEcho** [comment]
caesura doesn't cross post sources from one indexer to another so only source torrents already on AnimeBytes will be transcoded and uploaded to AnimeBytes.

I haven't tested to see if the AnimeBytes API has any quirks that make it incompatible with the OPS or RED implementation, and don't presently have the capacity to test. I'll leave this open as an **idea** though to see if there is demand.

- [ ] Test compatibility with AnimeBytes API
